docs/asciidoc-writers-guide.adoc

```diff
@@ -366,6 +366,8 @@ include::{includedir}/xref.adoc[]
 
 include::{includedir}/xref-relative.adoc[]
 
+TIP: The xref inline macro by default resolves the link to HTML, to create a relative link to an asciidoc file in another subdirectory include `ifdef::env-github[:outfilesuffix: .adoc]` below the document header and drop the ".adoc" extension after the filename.
```

Comment:
I'd like to see outfilesuffix set to `.adoc` by default on GitHub. There's an issue tracking that change. See https://github.com/github/markup/issues/519